modules/dcache-chimera/src/main/java/org/dcache/chimera/namespace/ChimeraCleaner.java

```diff
@@ -75,6 +75,9 @@
     private static final Logger _log =
         LoggerFactory.getLogger(ChimeraCleaner.class);
 
+    // ToDo: Spring Implementation for queryLimit ( used in runNotification() )
+    private final int queryLimit = 5000;
```

Comment:
As this is a constant for now, please make is **private final static**. And constants should be UPPER case.